testsuite/tests/misc/event_handler/main.adb

```diff
@@ -0,0 +1,35 @@
+with Ada.Text_IO;    use Ada.Text_IO;
+
+with Langkit_Support.Text; use Langkit_Support.Text;
+
+with Libfoolang.Analysis;  use Libfoolang.Analysis;
+
+with Support; use Support;
+
+procedure Main is
+   Ctx : Analysis_Context;
+
+   Dummy_Unit : Analysis_Unit;
+
+begin
+   Put_Line ("main.adb: Starting...");
+
+   --  Create a context with our file reader
+
+   declare
+      My_Sig_Handler : constant Event_Handler_Reference :=
+         Create_Event_Handler_Reference
+           (Sig_Handler'(null record));
```

Comment:
This probably won’t build (`Event_Handler`)

There are other occurences (also missing message updates)